### ISSUE TITLE
support for "allow for session"

### ISSUE THREAD

**Issue by fstanchina** [craigmiskell/cookiemaster#16]
Nice work, except for one detail -- it doesn't support the "allow for session" setting and that's the whole reason I want this: have the default Firefox setting at "allow for session" and change to "allow" or "block" for some specific domains.
Any chance you could implement that?

**Comment by craigmiskell** [comment]
Yes; I have most of that code written on a branch but got diverted by a bunch of other changes.  It's definitely something I want to do, just a matter of time.

**Comment by craigmiskell** [comment]
Or something similar to that, perhaps.
